messages/progress-event-listener.md

```diff
@@ -0,0 +1,18 @@
+# selection-spinner.action
+*DRAFT*: Selecting rules
+
+# selection-spinner.status
+*DRAFT*: Eligible engines: %s; Completion: %d%; Elapsed time: %ds
+
+# execution-spinner.action
+*DRAFT*: Executing rules
```

Comment:
The old version of this message was "executing X rules across Y engines". But the listener doesn't know how many engines there are anymore, so I removed that information.